docs/pages/build-reference/infrastructure.mdx

```diff
@@ -183,11 +183,13 @@ Android builders run on virtual machines in an isolated environment. Every build
 
 ## iOS build server configurations
 
-iOS builder VMs run on Mac Mini hosts in an isolated environment. Every build gets its own fresh macOS VM. For more information, see [see iOS-specific resource class](/eas/json/#resourceclass-2).
+iOS builder VMs run on Mac mini hosts in an isolated environment. Every build gets its own fresh macOS VM. For more information, see [iOS-specific resource classes](/eas/json/#resourceclass-2).
```

Comment:
```suggestion
iOS builder VMs run on Mac Mini hosts in an isolated environment. Every build gets its own fresh macOS VM. For more information, see [iOS-specific resource classes](/eas/json/#resourceclass-2).
```

Reply:
I too was surprised!! https://www.apple.com/mac-mini/ 

<img width="662" alt="Zrzut ekranu 2023-12-15 o 20 19 53" src="https://github.com/expo/expo/assets/1151041/2a50d42e-8d95-4ab5-a3e1-c70520b1b04a">

Reply:
The mini is intentionally lowercased (same with Apple silicon)